regression/goto-instrument-unwind/unwind-chain.sh

```diff
@@ -6,22 +6,18 @@ goto_instrument=$src/goto-instrument/goto-instrument
 cbmc=$src/cbmc/cbmc
 
 function usage() {
-  echo "Usage: chain unwind-num test_file.c"
+  echo "Usage: chain <options> <testfile>"
   exit 1
 }
 
-echo "Usage: process test_file.c to goto program"
-
-if [ $# -eq 2 ]
-then
-  unwind_num=$1
-  name=`echo $2 | cut -d. -f1`
-  echo $unwind_num
-  echo $name
-else
+if [ $# -ne 2 ]; then
   usage
+  exit 1
 fi
 
+name=$(echo $2 | cut -d. -f1)
```

Comment:
To address both the argument swapping and add support for path names containing "." use:
```
name=$(echo $1 | sed 's/\.[^\.]*$//')
shift
```